app/javascript/article-form/components/Help/Format.jsx

```diff
@@ -0,0 +1,135 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+const Format = ({ show, toggleModal }) => {
+  if (show) {
```

Comment:
refactor (blocking): For a few of the components that do `if (show) { ...` to render, move the responsibility of showing or hiding the component to the parent component.